src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -1072,10 +1180,120 @@ inline void rpp_store48_f32pln3_to_f16pkd3_avx(Rpp16f *dstPtr, __m256 *p)
     _mm_storeu_si128((__m128i *)(dstPtr + 45), px128[3]);
 }
 
+inline void rpp_load96_u8pln3_to_f32pln3_avx512(Rpp8u *srcPtrR, Rpp8u *srcPtrG, Rpp8u *srcPtrB, __m512 *p)
+{
+    __m512i px[6];
+    px[0] = _mm512_loadu_si512((__m512i *)srcPtrR);
```

Comment:
There are too many load and extract functions used here
Please check if you can directly use sse load instruction
_mm_loadu_si128 and finally fit into 512 vector